kit/ChildSession.cpp

```diff
@@ -2381,6 +2386,18 @@ bool ChildSession::resetSelection(const StringVector& tokens)
     return true;
 }
 
+namespace
+{
+static std::string extractFormat(const std::string& path)
```

Comment:
Surely we have this already. At least we have `Util::split()` and `Util::splitLast()` that make the implementation trivial, and in that case, consider having this in FileUtil, please, with a more meaningful name like `extractFileExtension` ("format" and "extension" aren't the same, of course, since a file can have the wrong extension).

See `Util::splitUrl()` for a very similar function.

Reply:
done